commit 93569c3e879d37442102d882084c58aab896fe66
Author: Tyler Hicks <tyhicks@canonical.com>
Date:   Tue Apr 5 12:35:23 2016 -0500

    UBUNTU: SAUCE: apparmor: Fix FTBFS due to bad include path
    
    When header files in security/apparmor/includes/ pull in other header
    files in that directory, they should only include the file name. This
    fixes a build failure reported by Tycho when using `make bindeb-pkg` to
    build the Ubuntu kernel tree but, confusingly, isn't seen when building
    with `fakeroot debian/rules binary-generic`.
    
    Signed-off-by: Tyler Hicks <tyhicks@canonical.com>
    Reported-by: Tycho Andersen <tycho.andersen@canonical.com>
    Cc: John Johansen <john.johansen@canonical.com>
    Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
    Signed-off-by: Leann Ogasawara <leann.ogasawara@canonical.com>

diff --git a/security/apparmor/include/domain.h b/security/apparmor/include/domain.h
index 89cfa75..b589655 100644
--- a/security/apparmor/include/domain.h
+++ b/security/apparmor/include/domain.h
@@ -15,7 +15,7 @@
 #include <linux/binfmts.h>
 #include <linux/types.h>
 
-#include "include/label.h"
+#include "label.h"
 
 #ifndef __AA_DOMAIN_H
 #define __AA_DOMAIN_H
